vispy/visuals/filters/mesh.py

```diff
@@ -721,8 +721,10 @@ def _update_data(self):
         self.fshader['width'] = self._width
         self.fshader['wireframe_only'] = 1 if self._wireframe_only else 0
         self.fshader['faces_only'] = 1 if self._faces_only else 0
-        faces = self._visual.mesh_data.get_faces()
-        n_faces = len(faces)
+        if self._visual.mesh_data.is_empty():
```

Comment:
Did this need to be changed? Did this work as expected before?

Reply:
This was broken because `get_faces` returns `None` when `Mesh` has no data. It was either this, or checkinf if `faces is None`, so I went for the method just implemented for clarity.

The test I added for `WireframeFilter` does not pass on main, currently.

Reply:
Ah awesome. Thanks.